terraform/modules/ecs/cluster/main.tf

```diff
@@ -0,0 +1,40 @@
+provider "aws" {
+  region = "us-east-1"
+  default_tags {
+    tags = module.platform.default_tags
+  }
+}
+
+provider "aws" {
+  alias  = "secondary"
+  region = "us-west-2"
+
+  default_tags {
+    tags = module.platform.default_tags
+  }
+}
+
+module "platform" {
+  source      = "github.com/CMSgov/cdap//terraform/modules/platform"
+  app         = var.app
+  env         = var.env
+  root_module = "https://github.com/CMSgov/cdap/tree/main/terraform/modules/ecs"
+  service     = "fargate"
+  providers   = { aws = aws, aws.secondary = aws.secondary }
+}
```

Comment:
The platform module should be referenced and passed into this module as a variable in the root module (service). See how this was done at https://github.com/CMSgov/cdap/blob/a59e580569f509e268d9368125c7326dee90ab72/terraform/modules/sops/variables.tf#L1